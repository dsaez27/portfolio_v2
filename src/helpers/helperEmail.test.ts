const helperEmail = require("./helperEmail")
// @ponicode
describe("helperEmail.validateEmail", () => {
    test("0", () => {
        let result = helperEmail.validateEmail("something@example.com")
        expect(result).toMatchSnapshot()
    })

    test("1", () => {
        let result = helperEmail.validateEmail("bed-free@tutanota.de")
        expect(result).toMatchSnapshot()
    })

    test("2", () => {
        let result = helperEmail.validateEmail("test.test@[0000.0.0.0]")
        expect(result).toMatchSnapshot()
    })

    test("3", () => {
        let result = helperEmail.validateEmail("foo.foo@\".AA")
        expect(result).toMatchSnapshot()
    })

    test("4", () => {
        let result = helperEmail.validateEmail("user1+user2@mycompany.com")
        expect(result).toMatchSnapshot()
    })

    test("5", () => {
        let result = helperEmail.validateEmail(undefined)
        expect(result).toMatchSnapshot()
    })
})
