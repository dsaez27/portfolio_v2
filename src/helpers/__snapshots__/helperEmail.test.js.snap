// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`helperEmail.validateEmail 0 1`] = `true`;

exports[`helperEmail.validateEmail 1 1`] = `true`;

exports[`helperEmail.validateEmail 2 1`] = `false`;

exports[`helperEmail.validateEmail 3 1`] = `false`;

exports[`helperEmail.validateEmail 4 1`] = `true`;

exports[`helperEmail.validateEmail 5 1`] = `false`;
